templates/getSecrets/getSecrets.yml

```diff
@@ -5,10 +5,14 @@ parameters:
 
 
 steps:
-  - ${{ if ne(parameters.KeyVaultName, '') }}:
```

Comment:
Was not sure how to write this one-liner with two conditions as it's done on line 14